internal/sbi/server.go

```diff
@@ -0,0 +1,142 @@
+package sbi
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/free5gc/openapi/models"
+	"github.com/free5gc/udr/internal/logger"
+	"github.com/free5gc/udr/internal/sbi/processor"
+	"github.com/free5gc/udr/internal/util"
+	"github.com/free5gc/udr/pkg/app"
+	"github.com/free5gc/udr/pkg/factory"
+	"github.com/free5gc/util/httpwrapper"
+	logger_util "github.com/free5gc/util/logger"
+)
+
+type Server struct {
+	app.UdrApp
```

Comment:
`UDR` you define in Line#30 
```suggestion
UDR
```